target/linux/bmips/bcm6318/base-files/etc/board.d/01_leds

```diff
@@ -6,7 +6,8 @@
 board_config_update
 
 case "$(board_name)" in
-comtrend,ar-5315u)
+comtrend,ar-5315u |\
+tp-link,td-w8968-v3)
 	ucidef_set_led_usbport "usb" "USB" "green:usb" "usb1-port1" "usb2-port1"
 	;;
```

Comment:
could we also setup the 'green:internet' here pointing at the WAN port?

Reply:
@wryun since the port is labelled both as LAN and WAN I think that we can leave it unconfigured and let the user decide the LED trigger.
![TD-W8968](https://static.tp-link.com/res/images/products/large/TD-W8968(UN)3.0-04.jpg)